src/app/tabs/run-tab.js

```diff
@@ -312,6 +313,12 @@ function contractDropdown (events, appAPI, appEvents, instanceContainer) {
     noInstancesText.style.display = 'none'
     var contractNames = document.querySelector(`.${css.contractNames.classNames[0]}`)
     var address = atAddressButtonInput.value
+    if (!ethJSUtil.isValidAddress(address)) {
+      return modalDialogCustom.alert('Invalid address.')
+    }
+    if (/[a-f]/.test(address) && /[A-F]/.test(address) && !ethJSUtil.isValidChecksumAddress(address)) {
```

Comment:
You could simplify the regexp into one: `/[a-fA-F]/`

Reply:
@axic -- We'd want to identify if an address consists both lowercase as well as uppercase characters and then only do the validation of checksum address. In case of the above suggested regex, if an address is comprised of either lowercase or uppercase characters will also be matched for which we wouldn't need the validation of checksum address as per the request by @yann300